### DISCUSSION TITLE
blog/indie-blogs

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#7] [Announcements]
# blog/indie-blogs

在折腾了将近一个月的博客后，我开始考虑要不要彻底关掉。

https://rercel.com/blog/indie-blogs

**Comment by ghost** [comment]
全文都是我心底话，有没有兴趣交笔友。


> 在折腾了将近一个月的博客后，我开始考虑要不要彻底关掉。

很高兴看到，后来还是坚持下来了。

> 记录个人生活没有动力

因为严肃话题的关键词缘故，不会有人专门搜，不少独立博客已经成了“独立的qq空间/朋友圈”。

对于我个人而言，写博客主要还是记录自己想热爱的，这是无价的。

其实现在读者也越来越功利，看知乎是如何发展到一个流量大站，大站都难免商业化、重营销。

另一方面，现在写博客非常需要feedback一种反馈，我推荐我的[一篇blog](https://www.xushilu.com/posts/blogging-is-not-an-obsolete-medium/)，对于这个媒介还是环境怎么批判也好，但个人的发展和爱好是不会变的，还记得当时写下第一篇blog的初心吗QuQ。

**Reply by Pathsis** [reply]
谢谢你的回响！很乐意与你做笔友！

这个博客不但要坚持下来，还会再写二十年😄

我同意你的想法，“写博客主要还是记录自己想热爱的，这是无价的”，不过，最近有了一个“博客病”的想法，有空会写出来。

共勉！

**Comment by DayuGuo** [comment]
不知不觉读了很久您的博客，受益匪浅。

**Reply by Pathsis** [reply]
谢谢欣赏。

其实我比你更早就通过推特了解了你的博客，博客设计赏心悦目！
